components/com_fabrik/models/form.php

```diff
@@ -3215,7 +3215,7 @@ public function getData()
 				}
 				else
 				{
-					$qs_request[$key] = $value;
```

Comment:
$qs_request is used later (line 3384) in case or rowid=-1 so I think it must be set, too.